features/tasks/createTask.feature

```diff
@@ -62,3 +62,24 @@ Feature: Set of tests to create tasks
 		Examples:
 			| userRole | courseName   | taskName             | studentUsername             		| studentPassword | taskBody  |
 			| teacher  | German       | private task example | amelia.strobl.qa@schul-cloud.org | Schulcloud1qa!  | task body |
+
+	@showTaskInFinishedTab @e2eCore
+	Scenario Outline: As a user, I want to be able to create a task and mark them completed
```

Comment:
Not sure if the scenario should stay that way, just to check if a task is properly moved. Seems a bit too much for that.